decibel/theme.json

```diff
@@ -32,7 +32,7 @@
 			]
 		},
 		"layout": {
-			"contentSize": "664px",
+			"contentSize": "1195px",
```

Comment:
I'm not sure about this, it's what the design implies but is quite a wide width from a readability perspective. Also should there be any difference between this and the wideSize? 

cc @henriqueiamarino

Reply:
I'm gonna leave it at 664 for now.   I definitely think there should be a difference between the two.  I'll assume that content was just wide width.